Modules/Sources/Networking/Remote/SystemStatusRemote.swift

```diff
@@ -16,19 +24,17 @@ public class SystemStatusRemote: Remote {
     ///
     public func loadSystemInformation(for siteID: Int64,
                                       completion: @escaping (Result<SystemStatus, Error>) -> Void) {
-        let path = Constants.systemStatusPath
-        let parameters = [
-            ParameterKeys.fields: [ParameterValues.environment, ParameterValues.activePlugins, ParameterValues.inactivePlugins]
-        ]
-        let request = JetpackRequest(wooApiVersion: .mark3,
-                                     method: .get,
-                                     siteID: siteID,
-                                     path: path,
-                                     parameters: parameters,
-                                     availableAsRESTRequest: true)
-        let mapper = SystemStatusMapper(siteID: siteID)
-
-        enqueue(request, mapper: mapper, completion: completion)
+        Task { @MainActor in
+            do {
+                let mapper = SystemStatusMapper(siteID: siteID)
+                let systemStatus = try await loadSystemStatus(for: siteID,
+                                                              fields: [Field.environment, Field.activePlugins, Field.inactivePlugins],
```

Comment:
Currently unused, but I think we're missing the settings field here?

```suggestion
                                                              fields: [Field.environment, Field.activePlugins, Field.inactivePlugins, Field.settings],
```

Reply:
`SystemStatusMapper`/[`SystemStatus`](https://github.com/woocommerce/woocommerce-ios/blob/e68c5800ff8b9b30efff965662998a2810566578/Modules/Sources/Networking/Model/SystemStatus.swift#L3-L5) currently does not need the settings field for the use case of this remote method (as in the fields in the [deleted line 21](https://github.com/woocommerce/woocommerce-ios/pull/15906/files#diff-0f22f2d30dde0fc3dbb3d14978bea91caa0329c6303a9d500ff58fe61eb756edL21)), thus I don't think we need to fetch it for performance reason.